airbyte-server/src/main/java/io/airbyte/server/handlers/SourcesHandler.java

```diff
@@ -96,6 +106,17 @@ public SourceRead updateSource(SourceUpdate sourceUpdate) throws ConfigNotFoundE
     return buildSourceRead(newSource);
   }
 
+  private void assertDockerImageIsValidIntegration(String dockerRepo, String tag) {
```

Comment:
This code is repeted

Reply:
Good idea, i was just thinking of how to get around this.

Reply:
Added an image validator and deduped tests